core/document/src/lib.rs

```diff
@@ -16,12 +20,32 @@ impl SvgElement {
 }
 
 #[derive(Default, Debug, Clone, PartialEq)]
+pub struct Folder {
+    elements: Vec<SvgElement>,
```

Comment:
Would a single Vec of tuples be better?

Reply:
Cache efficiency Data oriented programming sorts of stuff